docs/operations/export_api.md

```diff
@@ -252,7 +252,7 @@ status:
 #### Format types
 There are 4 format types that are possible:
 
-* Raw. The unaltered raw KubeVirt disk image.
+* Raw. The unaltered raw KubeVirt disk image. Raw images may be larger than expected if they were previously resized to match the PVC requested size.
```

Comment:
I would note something about the virtual size vs the actual size. A standard ls will report the virtual size of a sparse image. This is why it might look larger, but won't really be that much larger when you look at the actual size

Reply:
That's right... However, since in practice we always show the actual size during the download, I guess that could lead to confussion if then we want to upload the image to a PVC with a smaller virtual size as described in https://issues.redhat.com/browse/CNV-36303 . I agree with you though, I'm not sure if this is the right documentation to specify this. 

What about something like this:

```md
* Raw. The unaltered raw KubeVirt disk image. Raw images' virtual size may be larger than expected if they were previously resized to match the PVC requested size.
```

Reply:
The reason I don't like these type of messages is that even if we uploaded it to a PVC of the exact size the virtual size was, and thus nothing changed. ls will still show the virtual size which is not the actual size. This is all perfectly normal behavior, so pointing it out seems pointless.